pycbc/filter/resample.py

```diff
@@ -177,6 +177,9 @@ def resample_to_delta_t(timeseries, delta_t, method='butterworth'):
 
 _highpass_func = {numpy.dtype('float32'): lal.HighPassREAL4TimeSeries,
                  numpy.dtype('float64'): lal.HighPassREAL8TimeSeries}
+_lowpass_func = {numpy.dtype('float32'): lal.LowPassREAL4TimeSeries,
+                 numpy.dtype('float64'): lal.LowPassREAL8TimeSeries}
+
```

Comment:
I suggest you remove this change now that it has been split to its own PR, to avoid conflicts when rebasing.

Reply:
These edits have now been removed